webpack.config.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 module.exports = env=> {
-    return [
-        require('./config/webpack-dev.config'),
-        require('./config/webpack-prod.config')
-    ];
-};
\ No newline at end of file
+  return [
+    require(`./config/webpack-${env}.config`),
```

Comment:
This only includes one of the configs, where previously we had both dev + prod. Will have to check if the demo works fine?